### PR TITLE
independence day mysuc

### PR DESCRIPTION
**Pull request by boodaliboo**
### Intent of your Pull Request
adds american song for independence day
#### Changelog

:cl:  
rscadd: Adds a america music working this time 
/:cl:

### CODE REVIEW

**Comment by boodaliboo** [comment]
Will fix this in a few hours

**Comment by MegaEmpirical** [comment]
A bit late.

**Comment by boodaliboo** [comment]
are you fucking kidding me

**Comment by boodaliboo** [comment]
i broke every var somehow

**Comment by yacabo** [comment]
I haven't looked into the code to confirm this or not so I apologies for my ignorance, but isn't there supposed to be a `)` at the end and or is there even a connected `lobby_music` var?

**Comment by boodaliboo** [comment]
> I haven't looked into the code to confirm this or not so I apologies for my ignorance, but isn't there supposed to be a `)` at the end and or is there even a connected `lobby_music` var?

yeah thats one of the problems

File: code/modules/holiday/holidays.dm [hunk 1]
```diff
@@ -252,7 +252,9 @@
 	name = "Independence Day"
 	begin_day = 4
 	begin_month = JULY
-
+	lobby_music = list(
+		"https://www.youtube.com/watch?v=5uPoDNEn3I0", // americ
```
Comment (Amelia0010) [inline code/modules/holiday/holidays.dm]: This will be really grating once you get to the 3rd round, please add more songs.

**Review by Amelia0010 — CHANGES_REQUESTED** [review]

**Comment by boodaliboo** [comment]
Yeah I will I just couldn’t think of other songs give me a bit to add some. Meore

**Pull request merged**